client/starwhale/core/runtime/model.py

```diff
@@ -908,13 +919,18 @@ def _copy_src(
 
         logger.info("[step:copy-deps]start to copy pip/conda requirement files")
         ensure_dir(self.store.snapshot_workdir / RuntimeArtifactType.DEPEND)
-        for _fname in config.dependencies._conda_files + config.dependencies._pip_files:
+        for _fname in (
+            config.dependencies._conda_files
+            + config.dependencies._pip_files
+            + lock_files
+        ):
```

Comment:
Check duplication?  
In case the user has used a filename like `requirements-sw-lock.txt` or `conda-sw-lock.yaml`.

Reply:
ok